src/Transforms/Tools/MeasureEntropy.cs

```diff
@@ -43,9 +43,9 @@ await foreach (var block in input)
                     .Select(x => - Math.Log2(x) * x)
                     .Sum();
                 var size = block.Dictionary.Serialize().Length;
-                Console.WriteLine($"this many symbols {block.Stream.Length}");
+                Console.WriteLine($"Number of symbols: {block.Stream.Length}");
```

Comment:
There isn't a good system for logging yet, and frankly, I think it will only complicate things at this stage...